### PR TITLE
feat(virtualization): add evict subcommand

### PR DESCRIPTION
**Pull request by yaroslavborbat**
Add evict subcommand for d8 virtualization

### CODE REVIEW

File: internal/virtualization/cmd/lifecycle/migrate.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	"github.com/deckhouse/deckhouse-cli/internal/virtualization/templates"
+)
+
+func NewMigrateCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	lifecycle := NewLifecycle(Migrate, clientConfig)
+	migrate := string(Migrate)
+	cmd := &cobra.Command{
+		Use:     migrate + " (VirtualMachine)",
+		Short:   cases.Title(language.English).String(migrate) + " a virtual machine.",
```
Comment (mvasl) [inline internal/virtualization/cmd/lifecycle/migrate.go:17]: Why make this so complicated?

**Review by mvasl — APPROVED** [review]
LGTM

File: internal/virtualization/cmd/lifecycle/migrate.go (hunk 1 above)
Comment (yaroslavborbat) [reply internal/virtualization/cmd/lifecycle/migrate.go]: fix

File: go.mod [hunk 2]
```diff
@@ -4,7 +4,7 @@ go 1.23.1
 
 require (
 	github.com/Masterminds/semver/v3 v3.3.0
-	github.com/deckhouse/virtualization/api v0.0.0-20240326113112-979c4f4f17b9
+	github.com/deckhouse/virtualization/api v0.0.0-20241002083157-20295370a7bf
```
Comment (diafour) [inline go.mod:7]: Can we use semver tag? v0.14.0 for example?

Comment (yaroslavborbat) [reply go.mod:7]: mb

**Review by diafour — APPROVED** [review]

**Pull request merged**